src/bidiMapper/domains/network/NetworkStorage.ts

```diff
@@ -163,6 +163,11 @@ export class NetworkStorage {
     }
   }
 
+  /** Returns true iff there's at least one network request. */
```

Comment:
```suggestion
  /** Returns true if there's at least one network request. */
```

Reply:
Is it used somewhere?

Reply:
`iff` was intentional...but changed to `if` to simplify as requested.